### PR TITLE
Adds missing items in features list

### PR DESCRIPTION
**Pull request by mustafaiman**
Follows the order in https://hazelcast.org/clients/node-js/

https://github.com/hazelcast/hazelcast-nodejs-client/issues/340

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -445,26 +445,40 @@ Hazelcast Node.js client supports the following data structures and features:
 * Queue
 * Set
 * List
-* Multi Map
-* Replicated Map
-* Ringbuffer
+* MultiMap
+* ReplicatedMap
+* RingBuffer
```
Comment (furkansenharputlu) [inline README.md]: It should be `Ringbuffer`.

Comment (Serdaro) [inline README.md]: Following the IMDG language:

* MultiMap
* Replicated Map
* Ringbuffer

File: README.md [hunk 2]
```diff
@@ -445,26 +445,40 @@ Hazelcast Node.js client supports the following data structures and features:
...
+* Fail Fast on Invalid Configuration
+* SSL Support (requires Enterprise server)
+* Mutual Authentication (requires Enterprise server)
+* Authorization
 * Smart Client
 * Unisocket Client
-* Hazelcast Native Serialization
 * Lifecycle Service
-* SSL connection support (requires enterprise server)
+* Heartbeat Service
```
Comment (furkansenharputlu) [inline README.md]: Is it necessary for user ?

File: README.md [hunk 3]
```diff
@@ -445,26 +445,40 @@ Hazelcast Node.js client supports the following data structures and features:
...
 * Hazelcast Cloud Discovery
+* IdentifiedDataSerializable Serialization
+* Portable Serialization
+* Custom Serializers
+* Global Serializer
```
Comment (furkansenharputlu) [inline README.md]: `Custom Serialization`
`Global Serialization`

File: README.md [hunk 4]
```diff
@@ -445,26 +445,40 @@ Hazelcast Node.js client supports the following data structures and features:
...
-* Predicates
+* Entry Processor
+* Query (Predicates)
+* Paging Predicate
+* Built-in Predicates
+* Listener with Predicate
+* Fast Aggregations
+* Near Cache Support
+* Eventual Consistency Control
+* Declarative Config (JSON)
```
Comment (furkansenharputlu) [inline README.md]: Declarative Configuration (JSON)

File: README.md [hunk 5]
```diff
@@ -445,26 +445,40 @@ Hazelcast Node.js client supports the following data structures and features:
...
+* Programmatic Config
```
Comment (furkansenharputlu) [inline README.md]: Programmatic Configuration

**Comment by mustafaiman** [comment]
I had followed what was written in https://hazelcast.org/clients/node-js/ in terms of capitalization or word choice. I see you generally find that worse (me too). I'll just change all to your suggestions.

File: README.md (hunk 2 above)
Comment (mustafaiman) [reply README.md]: I dont think so. I just added it because it was a thing in client feature comparison.

File: README.md [hunk 6]
```diff
@@ -445,26 +445,39 @@ Hazelcast Node.js client supports the following data structures and features:
 * Queue
 * Set
 * List
-* Multi Map
-* Replicated Map
+* MultiMap
+* Replicate dMap
```
Comment (furkansenharputlu) [inline README.md]: `ReplicatedMap`

**Pull request merged**